osu.Game/Screens/SelectV2/Panel.cs

```diff
@@ -253,13 +248,20 @@ private void updateAccentColour()
 
             selectionLayer.Colour = ColourInfo.GradientHorizontal(backgroundColour.Opacity(0), backgroundColour.Opacity(0.5f));
 
-            updateEdgeEffect(animated: false);
+            updateSelectedState(animated: false);
         }
 
-        private void updateEdgeEffect(bool animated = true)
+        private void updateSelectedState(bool animated = true)
```

Comment:
I kinda don't get what this change does or what it's for but probably fine?

Reply:
The general change here is to make selected states more prominent.

- The edge glow is more refined and more like a border now (it's the best I could make it, still not happy)
- The edge glow and the colour gradient coming from the right of the panel now also applies to expanded beatmap set headers. Before they didn't really feel "selected" enough for my liking. This also applies to standalone panels.

Reply:
My main point of confusion is how it is the "selected" / "expanded" distinction. Is it possible even to have a beatmap set expanded but not selected? Or am I getting confused visually by this change?

Reply:
The selection change is for standalone panels, which are never expanded but are selected. The change makes this work there too.

(also applies to selected `PanelBeatmap` aka difficulties in grouped mode)